packages/smooth_app/lib/pages/folksonomy/tag.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class Tag extends StatelessWidget {
+  const Tag({super.key, required this.text});
+  final String text;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 4.0),
```

Comment:
Could you replace `4.0` with `VERY_SMALL_SPACE`?
+ generally speaking, avoid the `EdgeInsets` class and prefer `EdgeInsetsDirectional` (supports LTR/RTL)